src/pages/BookDetail.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
```

Comment:
I have imported the FA icons directly from the FA library, which is a bit more user-friendly.
However, the getStars() function does not work properly yet.

Maybe we should finish this functionality before we merge it with master.

Reply:
@riaars I've fixed this.
What else do you need to do? And what kind of help may I offer?

Reply:
@Envl where can we see that you have fixed this?

Reply:
@bt-s I've pushed to this branch. Just check it at /detail page.